cclib/method/moments.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2018, the cclib development team
+#
+# This file is part of cclib (http://cclib.github.io) and is distributed under
+# the terms of the BSD 3-Clause License.
+
+"""Calculation of electric multipole moments based on data parsed by cclib."""
+
+import numpy
+
+from cclib.parser.utils import convertor
+from cclib.method.calculationmethod import Method
+
+
+class Moments(Method):
```

Comment:
So, we parse moments already from logfiles, when they are present. The method is useful for situations where moments are not printed (and calculating them at different COMs). Do you have any suggestions about how to expose these two things in a consistent way? I realize you may no have context for this - @berquist @ATenderholt any thoughts?

Reply:
Yes, this could potentially cause confusions during further data inspection or comparison. Personally, I see a few practical weaknesses with the idea of updating the data object: (1) does it make any sense to compare parsed results and calculated ones coming from available partial atomic charges; (2) how to determine which type of atomic charges is used to evaluate moments; (3) there is no way to check is it parsed or calculated values.

Reply:
1. Sure, why not? They might be a decent approximation if the partial charge scheme is good. However, you're correct that they're fundamentally different.

Because they're fundamentally different, plus your point 3, plus you might want to compare different partial charge schemes (Mulliken, Lowdin, GAPT, ChElPG, ...), it doesn't make sense to overwrite the parsed moments in`ccData`, but at most store them in the `Method` object.